### ISSUE TITLE
EM Artifact scans incorrectly

### ISSUE THREAD

**Issue by nguyentvan7** [frzyc/genshin-optimizer#2368]
![image](https://github.com/user-attachments/assets/1a9602b8-a654-451e-96d5-329a2aa88aad)
![image](https://github.com/user-attachments/assets/a838bfe1-ebe4-46ad-8a86-b0cebf95cf72)

**Comment by legndery** [comment]
OCR is not able to read the EM main stat value:
```
texts
(2) ['Sands of Eon\n', 'Elemental Mastery\n']
```

**Comment by frzyc** [comment]
You are right, I am not sure what we can do to make this more accurate. Perhaps instead of a b/w highpass, we apply a better filter in this section? I would imaging the semi-transparent text being harder to OCR
![image](https://github.com/user-attachments/assets/373699bb-f6da-4c84-8b02-5f47f0d7932e)

**Comment by legndery** [comment]
I tried few standard preprocessing steps and it seems the normal gaussian blur -> dilation -> invert -> threshold is working. We can crop the stars out to be more accurate though, with the histogram we are already doing it. 
![image](https://github.com/user-attachments/assets/d33f0f7f-7fa3-483b-88e6-54957475ff7b)
Result:
![image](https://github.com/user-attachments/assets/0703a76e-9300-42b8-9589-90f40173b7aa)

Ref taken from https://replit.com/@yuispl2/GameToText#script.js this as-is worked, corresponding article: https://dev.to/mathewthe2/using-javascript-to-preprocess-images-for-ocr-1jc

Is there any regression test we can run with you know bunch of images?
